docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -179,32 +179,5 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 To use csharp api for openvino execution provider create a custom nuget package. Two nuget packages will be created
 Microsoft.ML.OnnxRuntime.Managed and Microsoft.ML.OnnxRuntime.Openvino.
 
-1. Windows
+Detailed instructions for building nuget packages can be found [here](./../../Build.md)
```

Comment:
Since this section is for openvino, we should mention the openvino specific flags: --use_openvino $Device.

Reply:
at the very top of this page they refer to build.md for build instructions... They added this section because all these extra steps needed to be carried out to create an openvino specific nuget package. I updated the instructions with more details.